packages/web/package.json

```
@@ -18,9 +18,9 @@
   "scripts": {
     "design-tokens:build": "test ! -f ../design-tokens/esm/index.js && cd ../design-tokens && yarn build || true",
     "prebuild": "shx rm -rf dist && shx mkdir -p dist/scss && shx cp package.json README.md dist/ && shx cp -r src/scss dist/ && shx rm -rf dist/scss/**/__tests__",
-    "build": "npm-run-all --serial --continue-on-error design-tokens:build css js",
+    "build": "npm-run-all --serial --continue-on-error prebuild design-tokens:build css js",
```

As pointed out elsewhere, this could be `build:prepare` instead of `prebuild`.

```suggestion
    "build": "npm-run-all --serial --continue-on-error build:prepare design-tokens:build css js",
```

---

Fixed.